drivers/scheduler/openshift/openshift.go

```diff
@@ -330,13 +332,66 @@ spec:
 	return err
 }
 
+// getImageSha get Imgae sha
+func getImageSha(ocpVersion string) (string, error) {
+	downloadURL := fmt.Sprintf("%s/%s/%s", OpenshiftMirror,
+		ocpVersion, mdFileName)
+	request := netutil.HttpRequest{
+		Method:   "GET",
+		Url:      downloadURL,
+		Content:  "application/json",
+		Body:     nil,
+		Insecure: true,
+	}
+	log.Infof("URL %s", downloadURL)
+	content, err := netutil.DoRequest(request)
+	if err != nil {
+		log.Errorf("Failed to get Get content from  %s, error %v", downloadURL, err)
+		return "", err
```

Comment:
Don't log error here, just return the whole verbose error and log it where you call this func if needed
```
return "", fmt.Errorf("Failed to get Get content from %s, error %v", downloadURL, err)
```